credentialz/credentialz.proto

```diff
@@ -620,14 +645,24 @@ message CaPublicKeyRequest {
 message CaPublicKeyResponse {
 }
 
+// A request to generate a public/private key pair based on a given key
+// specification and return the public key back which will be used to create
+// a hsot SSH certificate.
+message GenerateKeysRequest {
+  KeyGen key_params = 1;
+}
+
+// Response from the target whether it can generate the key pairs
+// for the given key specification and the public key if it generated it.
+message GenerateKeysResponse {
+  bool can_generate = 1;
```

Comment:
There's this can_generate, but also if the message set for CanGenerate is kept, the can_generate is already answered there, right? Why would this need to exist in both places?

Reply:
The original intention was to indicate the failure if the requested key can't be generated (The requested key in CanGenerateKey and GenerateKeyRequest are different). We can return an error in such scenarios and don't need this additional field `can_generate`, hence removed it.